lib/increadtimeout/incrt.go

```diff
@@ -52,7 +52,7 @@ func (crt *incrt) Read(buf []byte) (int, error) {
 
 	err := crt.rd.SetReadDeadline(start.Add(crt.wait))
 	if err != nil {
-		log.Warnf("unable to set deadline: %+v", err)
+		log.Debugf("unable to set deadline: %+v", err)
```

Comment:
Do we have an issue open about it? Likely worth looking into at some point

Reply:
That is because during tests we are using `net.Pipe` which does not support deadlines.

Reply:
Yeah, its the tests using io.Pipe. You can't set a deadline on io.Pipe